src/test/java/picard/sam/markduplicates/MarkDuplicatesSetSizeHistogramTest.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2018 The Broad Institute
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package picard.sam.markduplicates;
+
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+
+/**
+ * This class defines the individual test cases to check the duplicate set histograms. 
+ * @author hogstrom@broadinstitute.org
+ */
+public class MarkDuplicatesSetSizeHistogramTest extends AbstractMarkDuplicatesCommandLineProgramTest {
+    @Override
+    protected MarkDuplicatesSetSizeHistogramTester getTester() {
+        return new MarkDuplicatesSetSizeHistogramTester();
+    }
+
+    @Test
+    public void TestSingleSet() {
+    // This tests checks that if I add two read pairs with the same alignment start, a duplicate
```

Comment:
Line indent is wrong. also missing spaces after `,`s below. I would just re-indent the file in the IDE to fix both of these issues, using `Code/Reformat Code`.